website/docs/api/large-language-models.mdx

```diff
@@ -302,6 +303,87 @@ max_n_words = 20
 path = "summarization_examples.yml"
 ```
 
+### EL (Entity Linking) {id="nel"}
+
+The EL links recognized entities (see [NER](#ner)) against a provided knowledge
+base (KB). The EL task prompts the LLM to choose a shortlist of (the most
+likely) candidates from the KB. How this KB looks like and stores data can be
+arbitrary.
+
+Note that the documents run through the entity linking task are expected to have
+recognized entities in `.ents`. This can be achieved by - prior to running the
+EL task - running the [NER task](#ner), a trained spaCy NER model or by setting
+your docs' entities manually.
```

Comment:
```suggestion
Note that the documents processed by the entity linking task are expected to have
recognized entities in their `.ents` attribute. This can be achieved by either running the 
[NER task](#ner), using a trained spaCy NER model or setting the entities manually prior
to running the EL task.
```